apps/main/src/aas/domain/submodel-base/table-extension.ts

```
@@ -0,0 +1,113 @@
+    return this.headerRow ? this.headerRow.getSubmodelElements() : [];
+  }
+
+  get rows(): ISubmodelElement[] {
+    return this.data.getSubmodelElements();
+  }
+
+  addColumn(column: ISubmodelElement, options?: TableModificationOptions): void {
+    if (!this.headerRow) {
+      this.addHeaderRow();
+    }
+    this.rows.forEach(row => row.addSubmodelElement(cloneSubmodelElement(column), options));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)